components/generic/LanguageSelector.vue

```diff
@@ -35,6 +35,13 @@
           return locale.code === this.$i18n.locale;
         });
       }
+    },
+    head() {
+      return {
+        htmlAttrs: {
+          lang: this.$i18n.locale
```

Comment:
This does not seem to change the attribute if I switch to another language.

To have the desired effect, we also need to remove from nuxt.config.js the lines https://github.com/europeana/portal.js/blob/master/nuxt.config.js#L17-L19

Reply:
It works fine for me, can you check again?

Reply:
it works only when there's a language selector on the page, as in it won't work without having `ENABLE_LANGUAGE_SELECTOR=1`